package.json

```diff
@@ -156,6 +171,7 @@
     "connected-react-router": "^6.4.0",
     "debug": "^4.0.1",
     "font-awesome": "^4.7.0",
+    "glob": "^7.1.4",
```

Comment:
In your PR, can you please remove `"glob": "^7.1.4"` from `devDependencies`, otherwise this PR introduces a duplicate package in `dependencies`.